inductiva/design.py

```diff
@@ -47,11 +52,6 @@ def explore_design_space(simulator: Simulator,
             input_file,
         )
 
-        if output_dir is None:
```

Comment:
Why remove this? Does files resolve path takes care of it?

Reply:
I just moved this to outside the loop.

Reply:
Ah sorry! Did not notice that at the start.